tests/test_interpolate.py

```diff
@@ -351,3 +352,623 @@ def test_duplicate_idxs_period_pairs():
+    5.1893    1.4260    0.0000 H   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  2  0  0  0  0
+  1  3  1  0  0  0  0
+  1  4  1  0  0  0  0
+M  END
+$$$$
+""",
+        removeHs=False,
+    )
+
+
+def get_inv_nitrogen():
+    return Chem.MolFromMolBlock(
+        """
+  Mrv2311 11042413493D
+
+  4  3  0  0  0  0            999 V2000
+   -0.0110   -0.0156    0.0270 N   0  0  0  0  0  0  0  0  0  0  0  0
+    1.0099    0.0080   -0.0138 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.3291   -0.4654   -0.8338 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.3291    0.9548   -0.0138 H   0  0  0  0  0  0  0  0  0  0  0  0
+  1  3  1  0  0  0  0
+  1  4  1  0  0  0  0
+  1  2  1  0  0  0  0
+M  END
+$$$$""",
+        removeHs=False,
+    )
+
+
+def get_non_inv_nitrogen():
+    return Chem.MolFromMolBlock(
+        """
+  Mrv2311 11042413503D
+
+  4  3  0  0  0  0            999 V2000
+   -0.0138   -0.0196    0.0339 N   0  0  2  0  0  0  0  0  0  0  0  0
+    1.2876    0.0101   -0.0174 F   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.4197   -0.5936   -1.0629 F   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.4197    1.2173   -0.0174 F   0  0  0  0  0  0  0  0  0  0  0  0
+  1  4  1  0  0  0  0
+  1  3  1  0  0  0  0
+  1  2  1  0  0  0  0
+M  END
+$$$$""",
+        removeHs=False,
+    )
+
+
+@pytest.mark.parametrize(
+    "mol_a, mol_b, core",
+    [
+        (get_aldehyde(), get_inv_nitrogen(), np.array([[0, 0], [1, 1], [2, 2], [3, 3]], dtype=np.int32)),
+        (get_inv_nitrogen(), get_aldehyde(), np.array([[0, 0], [1, 1], [2, 2], [3, 3]], dtype=np.int32)),
+    ],
+)
+def test_core_achiral_interpolation_sp2_to_invertible_sp3(mol_a, mol_b, core):
+    # test that we're using the default interpolation schedule spanning over the entirely of lambda [0,1]
+    # for achiral transformations.
+    ff = Forcefield.load_default()
+
+    st = SingleTopology(mol_a, mol_b, core, ff)
+
+    lhs = st.setup_intermediate_state(0.0)
+    rhs = st.setup_intermediate_state(1.0)
+
+    # check that chiral volume is undefined in both end-states
+    assert len(lhs.chiral_atom.potential.idxs) == 0
+    assert len(rhs.chiral_atom.potential.idxs) == 0
+
+    lhs_bond_k = lhs.bond.params[0][0]
+    lhs_bond_b0 = lhs.bond.params[0][1]
+    lhs_angle_k = lhs.angle.params[0][0]
+    lhs_angle_a0 = lhs.angle.params[0][1]
+
+    rhs_bond_k = rhs.bond.params[0][0]
+    rhs_bond_b0 = rhs.bond.params[0][1]
+    rhs_angle_k = rhs.angle.params[0][0]
+    rhs_angle_a0 = rhs.angle.params[0][1]
+
+    # no chiral conversion happens, so we should be using the full schedule for bonds and angles
+    atol_k = 1e-3
+    atol_b0 = 1e-5
+    atol_a0 = 1e-4
+    for lamb in np.linspace(0.05, 0.95, 12):
+        itm = st.setup_intermediate_state(lamb)
+        itm_bond_k = itm.bond.params[0][0]
+        itm_bond_b0 = itm.bond.params[0][1]
+        itm_angle_k = itm.angle.params[0][0]
+        itm_angle_a0 = itm.angle.params[0][1]
+
+        assert abs(itm_bond_k - lhs_bond_k) > atol_k
```

Comment:
nit: consider `pytest.approx` (or `np.testing.assert_allclose`, if vectorizing over all ixns) for readability and more useful failure diffs

Reply:
obsolete, re-using `_assert_bonded_term` for this now.